INSTALL.md

```diff
@@ -24,6 +24,8 @@ Fedora (copr) :<br/>
 `sudo dnf install g810-led`<br/>
 Gentoo :<br/>
 `emerge app-misc/g810-led`
+Debian :<br/>
+`apt install g810-led`
```

Comment:
Hello and thanks for the PR, but I've a comment ...
`apt install g810-led` only works for Unstable/SID.
It would be nice to say so, right?

Reply:
Indeed, fixed, thanks!